compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -133,6 +137,7 @@ fun TooltipArea(
             .onPointerEvent(PointerEventType.Enter) { startShowing() }
```

Comment:
Not sure, but may be sometimes mouse can enter the component and don't move?
So, let's update the cursorPosition here, and may be clear it in Exit?

Reply:
I'll set it on Enter, but I'm not sure I want to clear it on exit. If by some chance we show the tooltip after exiting, it would be (slightly) better to show it at the last mouse position, rather than at 0,0.